### DISCUSSION TITLE
about solve.status

### DISCUSSION THREAD

**Discussion by wagjixiang** [coin-or/pulp#583] [Q&A]
Hellow,
I had create a MIP model, and I used warm-start to solve and limited the solve time, but I found a problem whenever my solution's status is optimal, i know it is not optimal, just is feasible, so can you tell me about this problem?
Thank you very much!

**Comment by Pasuukki** [comment]
Hi @wagjixiang! I think you may be talking about something that I struggled a long time with. I eventually discovered that the pulp.LpProblem class has 2 different statuses:

- status = status of the **problem**. If this equals pulp.constants.LpStatusOptimal, it **does not yet mean** that the solution is optimal, but that a feasible solution has been found.
- sol_status = status of the solution. If this equals pulp.constants.LpSolutionOptimal, I think then the solution should be optimal (at least inside the tolerances that you have defined).

**Reply by wagjixiang** [reply]
OK! Thank you~
I'm almost forgetting that. I will check it next time.
